tests/acl/test_acl_l3v4v6.py

```diff
@@ -0,0 +1,1224 @@
+    "ipv6": "acltbv4v6_v6_test_rules.j2"
+}
+ACL_RULES_PART_TEMPLATES = {
+    "ipv4": tuple("acltbv4v6_test_rules_part_{}.j2".format(i) for i in range(1, 3)),
+    "ipv6": tuple("acltbv4v6_v6_test_rules_part_{}.j2".format(i) for i in range(1, 3))
+}
+
+DEFAULT_SRC_IP = {
+    "ipv4": "20.0.0.1",
+    "ipv6": "60c0:a800::5"
+}
+
+
+# TODO: These routes don't match the VLAN interface from the T0 topology.
+# This needs to be addressed before we can enable the v6 tests for T0
+DOWNSTREAM_DST_IP = {
+    "ipv4": "192.168.0.253",
+    "ipv6": "20c0:a800::14"
+}
+DOWNSTREAM_IP_TO_ALLOW = {
+    "ipv4": "192.168.0.252",
+    "ipv6": "20c0:a800::1"
+}
+DOWNSTREAM_IP_TO_BLOCK = {
+    "ipv4": "192.168.0.251",
+    "ipv6": "20c0:a800::9"
+}
+
+# Below M0_L3 IPs are announced to DUT by annouce_route.py, it point to neighbor mx
+DOWNSTREAM_DST_IP_M0_L3 = {
+    "ipv4": "192.168.1.65",
+    "ipv6": "20c0:a800:0:1::14"
+}
+DOWNSTREAM_IP_TO_ALLOW_M0_L3 = {
+    "ipv4": "192.168.1.66",
+    "ipv6": "20c0:a800:0:1::1"
+}
+DOWNSTREAM_IP_TO_BLOCK_M0_L3 = {
+    "ipv4": "192.168.1.67",
+    "ipv6": "20c0:a800:0:1::9"
+}
+
+# Below M0_VLAN IPs are ip in vlan range
+DOWNSTREAM_DST_IP_VLAN = {
+    "ipv4": "192.168.0.123",
+    "ipv6": "fc02:1000::5"
+}
+DOWNSTREAM_IP_TO_ALLOW_VLAN = {
+    "ipv4": "192.168.0.122",
+    "ipv6": "fc02:1000::6"
+}
+DOWNSTREAM_IP_TO_BLOCK_VLAN = {
+    "ipv4": "192.168.0.121",
+    "ipv6": "fc02:1000::7"
+}
+
+DOWNSTREAM_IP_PORT_MAP = {}
+
+UPSTREAM_DST_IP = {
+    "ipv4": "194.50.16.1",
+    "ipv6": "20c1:d180::11"
+}
+UPSTREAM_IP_TO_ALLOW = {
+    "ipv4": "193.191.32.1",
+    "ipv6": "20c1:cb50::12"
+}
+UPSTREAM_IP_TO_BLOCK = {
+    "ipv4": "193.221.112.1",
+    "ipv6": "20c1:e2f0::13"
+}
+
+VLAN_BASE_MAC_PATTERN = "72060001{:04}"
+
+LOG_EXPECT_ACL_TABLE_CREATE_RE = ".*Created ACL table.*"
+LOG_EXPECT_ACL_TABLE_REMOVE_RE = ".*Successfully deleted ACL table.*"
+LOG_EXPECT_ACL_RULE_CREATE_RE = ".*Successfully created ACL rule.*"
+LOG_EXPECT_ACL_RULE_REMOVE_RE = ".*Successfully deleted ACL rule.*"
+
+PACKETS_COUNT = "packets_count"
+BYTES_COUNT = "bytes_count"
+
+
+@pytest.fixture(scope="module", autouse=True)
+def remove_dataacl_table(duthosts):
+    """
+    Remove DATAACL to free TCAM resources.
+    The change is written to configdb as we don't want DATAACL recovered after reboot
+    """
+    TABLE_NAME = "DATAACL"
+    for duthost in duthosts:
+        lines = duthost.shell(cmd="show acl table {}".format(TABLE_NAME))['stdout_lines']
+        data_acl_existing = False
+        for line in lines:
+            if TABLE_NAME in line:
+                data_acl_existing = True
+                break
+        if data_acl_existing:
+            # Remove DATAACL
+            logger.info("Removing ACL table {}".format(TABLE_NAME))
+            cmds = [
+                "config acl remove table {}".format(TABLE_NAME),
+                "config save -y"
+            ]
+            duthost.shell_cmds(cmds=cmds)
```

Comment:
Is it necessary to remove `DATAACL` to free TCAM?

Reply:
Yes. The intent of the test is to verify whether L3V4V6 ACL table is working as expected and NOT verifying the ACL table scale.